packages/expo-sms/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 
 ### 💡 Others
 
+- Converted Android code to Kotlin ([#13505](https://github.com/expo/expo/pull/13505) by [@kkafar](https://github.com/kkafar))
+
```

Comment:
There is no need to separate entries with newlines 😉 Also, just so you know — you don't have to add these links manually (this is inconvenient as you have to create the PR first and push once again). You can just add the note and `expo-ci` bot will give you a code review suggestion that you can apply from here

```suggestion
- Converted Android code to Kotlin ([#13505](https://github.com/expo/expo/pull/13505) by [@kkafar](https://github.com/kkafar))
```